.gitignore

```diff
@@ -1,3 +1,4 @@
+*.DS_Store
```

Comment:
I have always felt that OS and editor specific belong in each person's own global gitignore (checkout the [thoughtbot dotfiles global gitignore](https://github.com/thoughtbot/dotfiles/blob/master/gitignore), for example) rather than in the project's gitignore.

That said, this file already has `*.swp` in it, which is a vim thing.

Totally your call on whether you want to include it.

so I guess I don't feel that strongly one way or another

Reply:
I was wondering why it wasn't there. I have it in my global config but I was editing it home and it slipped I agree that specific settings should belong in your own environment. I'll remove it 👍